### ISSUE TITLE
input condition/testbed

### ISSUE THREAD

**Issue by aremazeilles** [nickkluft/udbenchmark_PIs#15]
Int he latest version of the documentation, we are suggesting to name the file containting the [tesbed configuration](https://eurobench.github.io/sofware_documentation/latest/data_format.html#_testbed_configuration_file) named `condition.yaml` instead of `testbed.yaml`.
The rationale is that the file may contain configuration that not only affect the testbed (as equipment), but also instruction given to the subject. 

I could change it easily within your repository here, but I am wondering if you could use similar terminology for the file your system generates (the code which generate your current file `cond_23_testbed.yaml`. 

A side question:   the filename is named `cond_23_testbed.yaml`. Doe sit mean that you have 22 other conditions before?

We also provide more suggestion for the datafile naming in [here](https://eurobench.github.io/sofware_documentation/latest/experiment_data.html). We would probably have to look at this as well later on.

**Comment by nickkluft** [comment]
I updated the scripts and my system now generates condition_##.yaml instead. 

Regarding you side question: we have in total 105 unique conditions. Which of the two namings should we use: condition_001.yaml or condition_01.yaml. 
Thanks for the provided information regarding the datafile naming.

**Comment by aremazeilles** [comment]
OK, I will adjust the repository content in that line, and get back to you for accepting that change.

If you can have 105 unique condition, then the filename should use 3 digit for the number. But are people likely to use in a singe experiment the 105 conditions?

**Comment by nickkluft** [comment]
Ok, I'll adjust that. The 105 conditions were all conditions that we did with one participant within one experiment. However, future studies might only select a specific set of conditions to test.

**Comment by aremazeilles** [comment]
I don't think you have to change anything in the code folder. Do you think the contrary?
The test data stored are fine as they are.
Eventually, we should add in the readme some indications on the data collected structure.
